installing/installing_gcp/installing-gcp-customizations.adoc

```diff
@@ -32,6 +32,8 @@ include::modules/installation-configuration-parameters.adoc[leveloffset=+2]
 
 include::modules/installation-gcp-config-yaml.adoc[leveloffset=+2]
 
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+
```

Comment:
Same here, this section is for IPI-on-GCP

Reply:
And for proxy support on GCP, we still have a testblocker bug https://bugzilla.redhat.com/show_bug.cgi?id=1753930, which was targeted to 4.3.

Reply:
openshift/installer#2405 is in the merge queue, which would unblock a 4.2 backport.  But yeah, restricting proxy GCP to UPI in the meantime would be a good, conservative choice.

Reply:
The 4.3/master changes all landed.  Hopefully we'll have the 4.2 backports soon.

Reply:
PRs for https://bugzilla.redhat.com/show_bug.cgi?id=1753930 have merged.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1753930 was already verified with 4.3 build. 
But if the 4.2 backports were not merged into 4.2.0, customer still have the problem when 4.2.0 is shipped.

Reply:
FYI - with PR #17080 we're limiting support to UPI installs only, so support is only for UPI for GCP now.

@danehans Were the PRs for https://bugzilla.redhat.com/show_bug.cgi?id=1753930 backported to 4.2, or is this still a problem in 4.2 for GCP UPI? 

If not, you mentioned there was a workaround of adding 'metadata.google.internal.' to noProxy - let me know if this is something that we need to document or not.

Reply:
@bergerhoffer The cloned bug for 4.2 https://bugzilla.redhat.com/show_bug.cgi?id=1759245 has been verified now, it's not an issue for 4.2.0 GCP UPI anymore, thanks!